### PR TITLE
Include I18n hints to README

### PR DESCRIPTION
**Pull request by fwolfst**

### CODE REVIEW

**Comment by excid3** [comment]
Wonderful! 🙏

File: README.md [hunk 1]
```diff
@@ -110,6 +110,24 @@ You can set a different partial name for calendars by passing the partial path.
...
+To profit from that, you can take advantage of the [`rails-i18n`](https://github.com/svenfuchs/rails-i18n/) gem which comes with translations for many languages already.
+
+In a Rails 6 app, the configuration could look like the following:
+
+  * Add `gem 'rails-i18n'` to your `Gemfile` and run `bundle`.
+  * Define the available and default locale e.g. in `config/application.rb`:
+```ruby  
+# config/application.rb
+config.i18n.available_locales = [:de, :fr]
+config.i18n.default_locale = :en
```
Comment (fwolfst) [inline README.md:126]: doh! `default_locale` has to be in `available_locales`!

**Comment by fwolfst** [comment]
Thanks for your timely responses and merges! I messed something up, though: https://github.com/excid3/simple_calendar/pull/250/files#r570549621 . `default_locale` should be in present `available_locales`. Could you fix that? Cheaper than a PR I think.

**Comment by excid3** [comment]
Got it! ✌️❤️

**Pull request merged**